### DISCUSSION TITLE
[Parseq example] Movement: horizontal panning, continuous

### DISCUSSION THREAD

**Discussion by rewbs** [rewbs/sd-parseq#94] [Parseq example library]
https://user-images.githubusercontent.com/74455/240471158-653df502-199a-4673-b2dc-5559dc23f475.mp4

Continuous horizontal camera panning, cubic spline interpolation. Small strength dips every 4 beats.

* 🚀 [Parseq direct link](https://sd-parseq.web.app/?importRemote=GUWZvNjsIQOWlB5HVb2uT3vEg4u1%2Fdoc-c9454057-623a-42b5-8706-fae98248cf63-1684896375771.json&token=4835dbda-cb5d-4204-8531-a7c5e7afdf2e)
* ⬇️ [Parsec doc](https://gist.github.com/rewbs/19a39424ad5973adf276de48f5f1164c)
* 📃 [Deforum settings](https://gist.github.com/rewbs/3675d5863de5ddb11b397d6a69dc94a4)
* ℹ️ Post processing notes: x3 FILM (20fps to 60fps)

**Comment by KoshiGlitch** [comment]
Love this example. 
I have done remix version with 110bpm, using ControlNet and doubled the pan, so it runs it twice. Used 0 as seed and init image from Midjourney.
I was hoping it would stick to that image for longer ;) or reveal again after panning back in. Like in your combining 3D y-axis rotation with x-axis pan.... not sure it is possible with one init. Here is [link to video](https://cdn.discordapp.com/attachments/1011415163913637961/1111335222471037039/20230525153819_FILM_x3.mp4) as 10MB limit on Github.
